.github/workflows/download-data.yml

```diff
@@ -20,4 +20,11 @@ jobs:
             - name: download schedule vs realtime
               uses: ./.github/actions/download-from-s3
               with:
-                file-path: schedule_vs_realtime_all_day_types_routes.json
\ No newline at end of file
+                file-path: schedule_vs_realtime_all_day_types_routes.json
+            - uses: oleksiyrudenko/gha-git-credentials@v2-latest
+              with:
+                token: '${{ secrets.GITHUB_TOKEN }}'
+            - run: |
+                git add .
+                git commit -m 'Add updated route data that was downloaded from s3'
+                git push origin save_downloaded_files
```

Comment:
Will this need to be changed to `git push origin main`?

Reply:
You are right. I had it saving to this feature branch for testing purposes, but I've now updated it to point to `main`.

Reply:
Great, thanks!